frontend/src/components/Tree/Tree.jsx

```diff
@@ -128,7 +132,16 @@ function renderTree(
         return TreeConf.VIEWING_NODE_SIZE;
       }
       return TreeConf.NODE_SIZE;
-    });
+    })
+    .on("mouseover", (d, i) => {
+      return tooltip
```

Comment:
yeah so basically in js, there are two kinds of anonymous functions:
(param) => {
       do stuff;
       return;
}


and (param) => return;

in the first one, if you put { }, then you have to return, but in the second one, it is returned automatically since it doesnt have a { }

Reply:
Can we standardize this though so we use the same format between both functions?

Reply:
Nice, looks like this code has been updated